erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -777,7 +777,7 @@ def add_additional_cost(stock_entry, work_order):
 		stock_entry.append('additional_costs', {
 			'expense_account': expenses_included_in_valuation,
 			'description': name[0],
-			'amount': items.get(name[0])
+			'amount': flt(items.get(name[0])) * flt(stock_entry.fg_completed_qty)
```

Comment:
The service item cost should be applied proportionately.

`service_item_cost * stock_entry.fg_completed_qty / work_order.qty`